.github/workflows/reviewdog.yml

```
@@ -0,0 +1,25 @@
+name: reviewdog
+on: [pull_request]
+jobs:
+  reviewdog:
+    name: reviewdog
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code.
+        uses: actions/checkout@v3.0.2
+
+      - name: Set up Go
+        uses: actions/setup-go@v3.2.0
+        with:
```

Maybe let's pin the version to 1.18.x
```suggestion
        with:
          go-version: "~1.18"
```

---

Why? Using the version specified in `go.mod` is much more conveniant.

---

😮  I was not aware of such a feature! Thank you :D